test/cli/test/browser-multiple.test.ts

```diff
@@ -11,6 +11,9 @@ it('automatically assigns the port', async () => {
   onTestFinished(() => spy.mockRestore())
   let ctx: Vitest
   let urls: (string | undefined)[] = []
+  // override default port since it can conflicts with
+  // other browser mode tests in test/cli suite
```

Comment:
test/cli runs sequentially, they should not affect each other

Reply:
I don't think it's sequential. I noticed config has thread options

https://github.com/vitest-dev/vitest/blob/25e6fe8a652777e61766a19f8951aff385739834/test/cli/vitest.config.ts#L8-L12

Is this meant to be `singleFork`? or `fileParallelism: false` is fine?

Reply:
Yes, it's supposed to make tests run sequentially